File: bench/json_parse.js
// json_parse — parse bench/data.json 20 times via V8 JSON.parse.
const fs = require("fs");
const path = require("path");
const src = fs.readFileSync(path.join(__dirname, "data.json"), "utf8");
let ok = 0;
for (let i = 0; i < 20; i++) {
    const obj = JSON.parse(src);
    if (obj !== null) ok++;
}
console.log(ok);

File: bench/json_parse.py
#!/usr/bin/env python3
# json_parse — parse bench/data.json 20 times.
import json, os
path = os.path.join(os.path.dirname(os.path.abspath(__file__)), "data.json")
with open(path, "rb") as f:
    src = f.read()
ok = 0
for _ in range(20):
    obj = json.loads(src)
    if obj is not None:
        ok += 1
print(ok)
